lib/header.c

```diff
@@ -241,6 +241,8 @@ Header headerFree(Header h)
 	}
 	h->index = _free(h->index);
     }
+    if (h->blob)
```

Comment:
Don't bother checking for non-NULL, free() does it anyway.

Reply:
Thank you for your comments. I've made changes.